srcpkgs/spnavcfg/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'spnavcfg'
+pkgname=spnavcfg
+version=1.1
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config qt5-host-tools"
+makedepends="libspnav-devel qt5-devel"
+depends="libspnav qt5-core qt5-gui qt5-widgets"
```

Comment:
some of these may be autodetected

Reply:
Waiting for feedback at https://github.com/void-linux/void-packages/pull/44959#discussion_r1259948835 and then I'll address this.

Reply:
Fixed in https://github.com/void-linux/void-packages/commit/a9e1cf978d63c9df85fd5014bef3ba18733af592.